### ISSUE TITLE
Not working with unauthenticated action cable connections

### ISSUE THREAD

**Issue by aurum-ellt** [basecamp/hotwire-spark#19]
Hello,

it doesn't work on the view `sessions/new` generated by the new authentication generator because it add this :

```
module ApplicationCable
  class Connection < ActionCable::Connection::Base
    identified_by :current_user

    def connect
      set_current_user || reject_unauthorized_connection
    end

    private
      def set_current_user
        if session = Session.find_by(id: cookies.signed[:session_id])
          self.current_user = session.user
        end
      end
  end
end
```

I fixed the problem removing this method `reject_unauthorized_connection` but i do not think it's a good solution.

**Comment by jorgemanrubia** [comment]
Aah thanks for reporting @aurum-ellt. 

I'll see how we can fix. I think the gem should patch ApplicationCable connection to skip auth when the requests come from hotwire spark.

**Comment by jorgemanrubia** [comment]
Released 0.1.8 with fix
